tests/conftest.py

```diff
@@ -70,7 +71,8 @@ def app(request):
         SECURITY_PASSWORD_SCHEMES=['plaintext'],
     )
     FlaskCLI(app)
-    FlaskCeleryExt(app)
+    celeryext = FlaskCeleryExt(app)
+    celeryext.celery.flask_app = app  # Make sure both apps are the same!
```

Comment:
❗ @lnielsen because of https://github.com/inveniosoftware/flask-celeryext/blob/master/flask_celeryext/app.py#L25-L26